### DISCUSSION TITLE
[Animation] What is the best practice for implementing a transition

### DISCUSSION THREAD

**Discussion by enkr1** [pixijs/pixijs#8100] [Q&A]
Hi community, I want to add animations (e.g. fade in & out, zoom in & out, etc.) to the elements such as `Container`, `Sprite`, etc.

I have been trying to do fade in and out by using [ticker](https://pixijs.download/dev/docs/PIXI.Ticker.html) which is simple, but I am not sure is it the best practice and if there are any existing packages I could use. 

Thank you so much for your help in advance.

---

- **`pixi.js` version**: _e.g. 4.7.1_
- **Browser & Version**: _e.g. Chrome 67_
- **OS & Version**: _e.g. Ubuntu 18.04_
- **Running Example**: _url_

**Comment by bigtimebuddy** [comment]
For general support questions, please use the discussions. Thank you for helping us to keep the project organized.

**Reply by themoonrat** [reply]
What may help you is a tween library, https://greensock.com/ being the most famous (but if this is a commercial project read the license first!), but there are plenty of others. They can automatically update a property of any object automatically, including properties like `alpha` to fade in and out certain containers.  And don't worry, that's best practice.... you can do all sorts of transitions just by moving / fading / scaling different containers.

**Reply by domis86** [reply]
or:
* https://github.com/pixijs/pixijs/discussions/7834#discussioncomment-1423998
* https://github.com/pixijs/pixijs/wiki/v5-Resources#tweens--animation

**Reply by enkr1** [reply]
Thank you so much @bigtimebuddy, @themoonrat & @domis86 for the solutions I will give them a try

I found a package which is quite simple to use! 

- https://github.com/srpatel/pixi-actions
